scripts/ci/pre_commit/pre_commit_revision_map.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+export PYTHON_MAJOR_MINOR_VERSION="3.7"
+export PRINT_INFO_FROM_SCRIPTS="false"
+
+# shellcheck source=scripts/ci/static_checks/revision_map.sh
+. "$( dirname "${BASH_SOURCE[0]}" )/../static_checks/revision_map.sh" "${@}"
```

Comment:
Do we need this wrapper?

Reply:
This is really historical "overhead" that over time through some changes became completely redundant. We should remove those.

The wrapers were only needed because we wnted to keep the possibility of running the checks separately via static_checks scripts (that was at the time we introduced pre-commits but some users wanted to keep the old ways of doing things). Pre-commits used to need some extra configuration in those wrappers, but a number of simplifications removed the need for those and those wrappers became empty-shells.. 

I willl remove thoose extra similar wrappers we have in a separate PR now.

Reply:
Pr here: https://github.com/apache/airflow/pull/21643 - let's kill some BASH